tests/scripts/optdata/bootstrap.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+
+"""
+    This script prepares the local source tree to be built with
+    custom optdata. Simply run this script and follow the
+    instructions to inject manually created optdata into the build.
+"""
+
+import argparse
+import json
+import os
+from os import path
+import shutil
+import subprocess
+import sys
+
+# Display the docstring if the user passes -h|--help
+argparse.ArgumentParser(description=__doc__).parse_args()
+
+SCRIPT_ROOT = path.dirname(path.realpath(__file__))
+REPO_ROOT = path.realpath(path.join(SCRIPT_ROOT, '..', '..', '..'))
+
+ORIGIN_FILE = path.join(SCRIPT_ROOT, 'project.json')
+TARGET_FILE = path.join(REPO_ROOT, 'src', '.nuget', 'optdata', 'project.json')
+
+ARCH_LIST = ['x64', 'x86']
+TOOL_LIST = ['IBC', 'PGO']
+
+def get_buildos():
+    """Returns the Build_OS component used by the build system."""
+    if os.name == 'nt':
+        return 'Windows_NT'
+    else:
+        sysname = os.uname()[0]
+        return 'OSX' if sysname.casefold() == 'Darwin'.casefold() else sysname
+
+def get_optdata_version(tool):
+    """Returns the version string specified in project.json for the given tool."""
+    package_name = 'optimization.{0}.CoreCLR'.format(tool)
```

Comment:
I don't really have a preference on this, but in other places, we've done this as

package_name = 'optimization.%s.CoreCLR' % (tool)

Likewise below

package_name = 'optimization.%s-%s.%s.CoreCLR' % (get_buildos(), arch, tool)